docs/layout-of-source-files.rst

```diff
@@ -35,9 +35,7 @@ a good idea to read through the changelog at least for releases that contain
 breaking changes. These releases always have versions of the form
 ``0.x.0`` or ``x.0.0``.
 
-The version pragma is used as follows::
-
-    pragma solidity ^0.5.2;
+The version pragma is used as follows: ``pragma solidity ^0.5.2;``
```

Comment:
Because this is the main example, I did not want to change it to something weird. instead, this is now not compiled anymore.

Reply:
What would something weird be?

Reply:
`>= 0.5.0 < 0.7.0`